webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -403,6 +404,16 @@ object RudderConfig extends Loggable {
     }
   }
 
+  val RUDDER_BATCH_CHECK_NODE_CACHE_INTERVAL = {
+    try {
+      Duration.fromScala(scala.concurrent.duration.Duration(config.getString("rudder.batch.check.node.cache.interval")))
+    } catch {
+      case ex: Exception =>
+        ApplicationLogger.info("Property 'rudder.batch.check.node.cache.interval' is missing or empty in rudder.configFile. Default to '15 s'.")
```

Comment:
shouldn't you add the value in the config file so that we don't have the log message?

Reply:
yes, right, even it technical, it will be more discoverable like that